certgen/csr_sign.c

```diff
@@ -27,20 +27,23 @@
 #include <wolfssl/wolfcrypt/asn.h>
 #include <wolfssl/wolfcrypt/error-crypt.h>
 
-#define HEAP_HINT NULL
-#define LARGE_TEMP_SZ 4096
-
 /* Check if the internal asn API's are available */
 #if defined(WOLFSSL_TEST_CERT) || defined(OPENSSL_EXTRA) || \
     defined(OPENSSL_EXTRA_X509_SMALL)
     #define HAVE_DECODEDCERT
 #endif
 
+#if defined(WOLFSSL_CERT_REQ) && defined(WOLFSSL_CERT_GEN) && \
+    defined(HAVE_ECC)
+
+#define HEAP_HINT NULL
+#define LARGE_TEMP_SZ 4096
+
 static void usage(void)
 {
-    printf("Usage: ./csr_sign [type] [csr.pem] [ca-cert.pem] [ca-key.pem]\n");
+    printf("Usage: ./csr_sign [csr.pem] [ca-cert.pem] [ca-key.pem]\n");
```

Comment:
I'm not positive about this, but I think square brackets means its optional.  But the way the code works below, these are not optional args.